### ISSUE TITLE
404 for http://localhost:48626/takana.js

### ISSUE THREAD

**Issue by patrikengborg** [mechio/takana#95]
localhost:48626 is giving me a 404 when requesting takana.js. "Cannot GET /takana.js" is the error when accessing it directly.

Everything seems fine and I get green output in the console with "webserver listening on 48626", "editor connected" and so on.

I had it running just a few days ago, but then something happened. Any ideas what the problem might be?

Node v5.1.1
npm v3.3.12

**Comment by damln** [comment]
Same here

node v4.2.0
npm v2.14.7
takana v0.5.2

**Comment by cholawo** [comment]
I had this issue and found that updating to npm 3.5.1 and trashing my project's node_modules and reinstalling made it work.

For reference, you're looking for a "rendered" message when takana has connected to the browser.

I lost at least a day trying to figure out why it wouldn't work when all I needed to do was be bold, update and reinstall my project's npm  (this didn't break anything else in my project, despite it having used npm 2 and being a couple years old). I think I also had to update gulp-sass to 2.1.1 so heads up if you use that.

It's annoying that there was no error shown from takana's side when this was happening, but it just wasn't connecting to the browser, which it was waiting for, so I guess it's not something it can catch as an error?
I noticed that npm 3 no longer installs dependencies as nested folders, so I speculate that that could have had something to do with it...

Additionally, I recommend using takana with gulp  - https://github.com/mechio/takana-example as it allowed me to specify the path of my sass files, which was another problem I was encountering,

**Comment by damln** [comment]
Thanks a lot @cholawo!
I will try that.

**Comment by patrikengborg** [comment]
Trashed my node_modules and installed everything from scratch. Still no luck though, same error as before.

Thanks @cholawo for trying to help out!

**Comment by damln** [comment]
Work for me.

    rm -rf node_modules/
    npm uninstall takana
    npm install -g takana
    node -v # v4.2.0
    npm -v # 3.5.2
    takana -V # 0.5.2

**Comment by cholawo** [comment]
Sorry to hear that @patrikengborg, did you definitely update your global npm version to 3.5.2?   The other thing you can try is when starting takana in command line add a ```-v``` and it will show you a bit more information.  Maybe you'll see a takana error if it really is being caused by something other than npm version.

**Comment by ptcc** [comment]
I get the same error... runing takana -v only gives the following output
[2016-02-03 00:14:07.661] [TRACE] Server - [::1] GET localhost:48626 /takana.js

**Comment by barnaby** [comment]
Ok, this was broken in https://github.com/mechio/takana/commit/0fe959d32fadcdfbbddeef93f6af45ac447d3a2e#diff-09d890aebbbde8c4607f54bd8e44b698.

@nc why did we switch to 'get-installed-path'? Can we revert to:
```
@app.use express.static(path.join(__dirname, '..', '/node_modules/takana-client/dist'))
```

**Comment by saiqulhaq** [comment]
I just tried @barnaby's suggestions and it works 
thanks @barnaby

**Comment by lukeb** [comment]
@barnaby Is this fixed yet in the NPM package?

**Comment by barnaby** [comment]
This is now fixed in v0.5.4

Apologies for the delay in fixing this.
